src/VisualStudio/Core/Def/Venus/ContainedLanguage.cs

```diff
@@ -34,7 +38,7 @@ internal partial class ContainedLanguage
 
         public VisualStudioProject? Project { get; }
 
-        protected readonly ContainedDocument ContainedDocument;
+        protected /*readonly*/ ContainedDocument ContainedDocument = null!; // Never actually null because the constructors initialize it.
```

Comment:
I was just cleaning things up since the constructors are basically identical except for how `filePath` is defined.

Reply:
Losing the readonly flag though is a bit unfortunate though -- I can't tell what really forced it though?

Reply:
Because the variable isn't set on the constructor. I'll follow @CyrusNajmabadi recommendation though and consolidate everything into a single private constructor.

Reply:
Yeah, that's what I was wondering too why we weren't doing a single constructor.